app/assets/stylesheets/components/_space-misc.scss

```diff
@@ -18,9 +18,13 @@
 .mtn1 { margin-top: -$space-1; }
 .mtn2 { margin-top: -$space-2; }
 .mtn3 { margin-top: -$space-3; }
+.mtn4 { margin-top: -$space-5; }
 
 @media #{$breakpoint-sm} {
   .sm-mr-20p { margin-right: 20px; }
+  .sm-mt0 { margin-top: 0; }
```

Comment:
This one exists already in `vendor/assets/stylesheets/basscss/_responsive-margin.scss`